traffic_ops/install/bin/postinstall.test.sh

```diff
@@ -15,18 +15,72 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
 set -e;
 
-if [[ ! -x /usr/bin/python3 ]]; then
+cd "$(dirname "${BASH_SOURCE[0]}")";
+readonly MY_DIR="$(pwd)";
+
+while getopts :23sb: opt; do
+	case "$opt" in
+		2)	python_version=2;;
+		3)	python_version=3;;
+		s)	skip_python2=true;;
+		b)	python_bin="$OPTARG";;
+		*)	echo "Invalid flag received: ${OPTARG}" >&2
+			exit 1;;
+	esac;
+done;
```

Comment:
I just realized; this used to have no options. Now that there _are_ options, it'd be nice to have a `-h` and/or `-?` to pass in order to see them.

Reply:
Added help string in 1e7fd5752d